database/migrations/20230818100709_create_users_table.sql

```diff
@@ -0,0 +1,16 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE "public"."users" (
```

Comment:
Question, why we have this `"public"`?

Reply:
I copied from the export database function 🙏